.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: generic
+
+services:
+- docker
+
+jobs:
+  include:
+  - stage: Build and Deploy
+    name: Build and Deploy
+    if: tag IS present
+    before_script: skip
+    script: make build "IMAGE_TAG=$TRAVIS_TAG"
+    deploy:
+      provider: releases
+      api_key: "$GITHUB_TOKEN"
+      skip_cleanup: true
+      overwrite: true
+      file_glob: true
+      file:
+      - _out/*
```

Comment:
You should copy something to _out first I guess..

Reply:
Removed